app/components/environment_banner_component.html.erb

```diff
@@ -0,0 +1,5 @@
+<header class="environment-banner">
+  <div class="govuk-width-container">
+    This is <%= Rails.env %>. Only process test cases on this version of BoPS
```

Comment:
I added this as I thought it might be used on other environments at some point. However, maybe I should just hardocde it for simplicity??